introduction/de/01-introduction.asciidoc

```diff
@@ -0,0 +1,12 @@
+== Einführung
+
+Diese Weiterbildungsserie bietet eine Einführung in InnerSource. 
+InnerSource ist die Anwendung von Open-Source Praktiken und -Prinzipien auf die Softwareentwicklung im Unternehmen. 
```

Comment:
```suggestion
InnerSource ist die Anwendung von Open-Source-Praktiken und -Prinzipien auf die Softwareentwicklung im Unternehmen. 
```

Reply:
I wonder how we should deal with words that are tightly coupled together. The options I see are...

1. Leave them as group (e.g. `Open Source`)
2. Connect them with hyphens (e.g. `Open-Source-Praktiken`)
3. Use compound words (e.g. `Opensourcepraktiken`).
4. Use compound words with unnatural capitalization (e.g. `DevOps`, `InnerSource`)

My thoughts on usage:
- Option (4), we should use only where it's the dominant way of writing for a specific term (e.g. we should only to it for `InnerSource`, I think)
- Option (3), quickly becomes clumsy, especially when German and English terms are mixed (e.g. `Opensourcepraktiken`). I think this should be avoided alltogether.
- **I'd suggest we always aim to use option (2), unless the term is established and widely known to be written like option (1).**

Suggestion: Change to `Open-Source-Praktiken und Prinzipien`

Reply:
Compound words sound fine to me.
Unless we want to play the popular German kids game "Who can make the longest compound word". I raise you a _"Rindfleischetikettierungsüberwachungsaufgabenübertragungsgesetz"_. ;)